Merit_Hydro_Visualization.js

```diff
@@ -0,0 +1,325 @@
+                              ':(b("viswth") > 3000) ? 15' +
+                              ':(b("viswth") > 2000) ? 14' +
+                              ':(b("viswth") > 1500) ? 13' +
+                              ':(b("viswth") > 1000) ? 12' +
+                              ':(b("viswth") > 700) ? 11' +
+                              ':(b("viswth") > 500) ? 10' +
+                              ':(b("viswth") > 300) ? 9' +
+                              ':(b("viswth") > 200) ? 8' +
+                              ':(b("viswth") > 150) ? 7' +
+                              ':(b("viswth") > 100) ? 6' +
+                              ':(b("viswth") > 70) ? 5' +
+                              ':(b("viswth") > 50) ? 4' +
+                              ':(b("viswth") > 20) ? 3' +
+                              ':(b("viswth") > 1) ? 2' +
+                              ':(b("viswth") == 0) ? 0' +
+                              ':1')
+                          .selfMask();
+
+var rwC_local = viswidthClass.updateMask(wth_mask_local);
+
+// Hand
+var handClass = hand
+                    .expression(
+                      '(b("hnd") > 300.0) ? 18' +
+                      ':(b("hnd") > 200.0) ? 17' +
+                      ':(b("hnd") > 150.0) ? 16' +
+                      ':(b("hnd") > 100.0) ? 15' +
+                      ':(b("hnd") > 70.0) ? 14' +
+                      ':(b("hnd") > 50.0) ? 13' +
+                      ':(b("hnd") > 30.0) ? 12' +
+                      ':(b("hnd") > 20.0) ? 11' +
+                      ':(b("hnd") > 15.0) ? 10' +
+                      ':(b("hnd") > 10.0) ? 9' +
+                      ':(b("hnd") > 7.0) ? 8' +
+                      ':(b("hnd") > 5.0) ? 7' +
+                      ':(b("hnd") > 2.0) ? 6' +
+                      ':(b("hnd") > 1.0) ? 5' +
+                      ':(b("hnd") > 0.7) ? 4' +
+                      ':(b("hnd") > 0.5) ? 3' +
+                      ':(b("hnd") > 0.1) ? 2' +
+                      ':1');
+
+//////////////////////////////////////////////////////////////
+// Set up the Interactive System on the Map
+//////////////////////////////////////////////////////////////
+
+// Set up the overall structure of the app, with a control panel to the left
+// of a full-screen map.
+ui.root.clear();
+var panel = ui.Panel({style: {width: '250px'}});
+var map = ui.Map();
+ui.root.add(panel).add(map);
+map.style().set('cursor', 'crosshair');
+
+// Create title
+panel.add(ui.Label(
+  'MERIT Hydro Visualization and Interactive Map',
+  {fontWeight: 'bold', fontSize:'20px'}
+));
+
+// Create a layer selector that dictates which layer is visible on the map.
+var select = ui.Select({
+  items: [RIVER_WIDTH, RIVER_CHANNEL, TERRAIN],
+  value: RIVER_WIDTH,
+  onChange: redraw,
+});
+panel.add(ui.Label('Use the Layers panel to choose which band values are shown on the map.')).add(select);
+
+// Check-boxes to control which layers are shown in the inspector.
+panel.add(ui.Label('Band Information:'));
```

Comment:
You can remove the rest of the panel now that you are recommending to turn layers on/off via the Layers control.

Or if you still want this to control which layers are queried when you click on the map, update the text to make it more clear.